src/config/docs.md

```diff
@@ -110,7 +110,7 @@ This page is automatically generated by `./risedev generate-example-config`
 | max_preload_wait_time_mill |  | 0 |
 | max_sub_compaction |  Max sub compaction task numbers | 4 |
 | max_version_pinning_duration_sec |  | 10800 |
-| mem_table_spill_threshold |  The spill threshold for mem table. | 0 |
+| mem_table_spill_threshold |  The spill threshold for mem table. | 4194304 |
```

Comment:
minor: Let's mention the unit of this config in the description: "The spill threshold measured in bytes for mem table"